### ISSUE TITLE
Telegram wallet and mini apps not working on Windows 11

### ISSUE THREAD

**Issue by NeoGeo2K** [telegramdesktop/tdesktop#27973]
### Steps to reproduce

1. Open Wallet
2. Or open any mini apps in TG
3. And "wait loading, wait, wait, wait, wait ..." 

### Expected behaviour

Wallet info. Mini apps games and etc.

### Actual behaviour

![wallet](https://github.com/telegramdesktop/tdesktop/assets/2789657/b41e61fc-2505-4821-bf6d-370803ca91a5)

### Operating system

Windows 11 Home

### Version of Telegram Desktop

5.0.2

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
Perhaps you're one of those folks messing with system files and then complaining that applications don't work as they supposed?

**Comment by MrCoder228re** [comment]
just install https://developer.microsoft.com/ru-ru/microsoft-edge/webview2

**Comment by NeoGeo2K** [comment]
> just install https://developer.microsoft.com/ru-ru/microsoft-edge/webview2

Thanks. It helped.

**Comment by john-preston** [comment]
Hmm, looks like WebView Legacy isn't at all reliable, maybe it should be disabled 🤔

**Comment by amlaku** [comment]
my telegram desktop can't open /work any airdrop/wallet what shall I do?

**Comment by ilya-fedin** [comment]
@amlaku reinstalling Windows should do the thing (webview not working is a sign your system is broken). Maybe SFC could also help but not sure.

**Comment by amlaku** [comment]
it loading but can't open even using sfc uninstall windows

**Comment by ilya-fedin** [comment]
Yeah, as I said, not surec about SFC. Reinstalling should help for sure.
